packages/client/tests/functional/issues/14373-batch-tx-error/tests.ts

```diff
@@ -0,0 +1,24 @@
+// @ts-ignore
+import type { PrismaClient } from '@prisma/client'
+
+import testMatrix from './_matrix'
+
+declare let prisma: PrismaClient
+
+testMatrix.setupTestSuite(() => {
+  test('correctly reports location of a batch error', async () => {
+    const result = prisma.$transaction([
+      prisma.user.findMany({}),
+      prisma.user.update({
+        where: {
+          email: 'notthere@example.com',
+        },
+        data: {
+          memo: 'id is 1',
+        },
+      }),
+    ])
+
+    await expect(result).rejects.toThrowError('Invalid `prisma.user.update()` invocation')
```

Comment:
Question: Does the error point at the line number of the error?

Reply:
Yes. Did not include full snapshot here because it differes for data proxy for some reason, but you can check interactive-transactions snapshots here - they now also point to correct query.